drivers/input/touchscreen/goodix.c

```diff
@@ -111,6 +111,8 @@ static const struct goodix_chip_id goodix_chip_ids[] = {
 	{ .id = "912", .data = &gt967_chip_data },
 	{ .id = "9147", .data = &gt967_chip_data },
 	{ .id = "967", .data = &gt967_chip_data },
+
```

Comment:
```suggestion
	{ .id = "917D", .data = &gt967_chip_data },
```
Our assumption for the firmware length was wrong (it’s 228 not 240 bits). With the change to &gt967_chip_data  `Error reading 1 bytes from 0x8140` vanishes.